builder/format.go

```diff
@@ -64,3 +68,63 @@ func RevertTrimmedGroupsINI(b *Builder) error {
 	return helpers.CopyFile(filepath.Join(b.Config.Builder.ServerStateDir, "groups.ini"), filepath.Join(b.Config.Builder.ServerStateDir, "trimmed_groups.ini"))
 }
 
+func getLastBuildVersion(b *Builder) (string, error) {
+	// Likely the first build or some weirdness that requires us to get the container image and build it anyway
```

Comment:
Does this comment refer to the error case below at L77?

Reply:
Yes it does.. had to fix some var shadowing and didn't move the comment.